lib/go-tc/deliveryservices.go

```diff
@@ -30,6 +30,13 @@ const DefaultMaxRequestHeaderBytes = 0
 const MinRangeSliceBlockSize = 262144   // 265Kib
 const MaxRangeSliceBlockSize = 33554432 // 32Mib
 
+// SSLKeysAddResponse is a struct to store the response of addition of ssl keys for a DS,
+// along with any alert messages
+type SSLKeysAddResponse struct {
```

Comment:
This should probably go in `deliveryservice_ssl_keys.go` instead and it looks like it can just use this type alias:
```
type SSLKeysAddResponse DeliveryServiceSSLKeysGenerationResponse
```
since `DeliveryServiceSSLKeysGenerationResponse` is the same.